src/jats/importer/jats-body-transformations.ts

```diff
@@ -194,19 +211,40 @@ export const jatsBodyTransformations = {
     references: BibliographyItem[] | null,
     createElement: (tagName: string) => HTMLElement
   ) {
+    const bodyContainer = createSectionContainer('body', createElement)
+    const abstractsContainer = createSectionContainer(
+      'abstracts',
+      createElement
+    )
+    const bodySections = doc.querySelectorAll(
+      'body > sec:not([sec-type="backmatter"]), body > sec:not([sec-type])'
```

Comment:
Is `sec[sec-type="backmatter"]` something that actually exist in the content?

Reply:
Yes. 
Before `moveSectionsToBody` we're calling `mapFootnotesToSections`. Which appends some of the `backmatter` sections to the body. So I created the `backmatter` container there so I don't have to query the sections again when moving sections to body.

Reply:
Can we call `mapFootnotesToSections` inside `moveSectionsToBody`? I don't see a reason why it's separate. This was we can create all three containers in the same place.

Reply:
Sure, I also did refactored `moveSectionsToBody` into multiple functions for better readability, let me know if you prefer the old code.